<!DOCTYPE html>
<html>
<head>
<title>COMICS-help</title>
<link href="../bin/yaz.css" rel="stylesheet" type="text/css">
<link rel="icon" href="../figures/yaz_lable.png" type="image/x-icon">
<meta http-equiv="Content-Language" content="en">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="viewport" content="width=640">
<meta name=viewport content="width=device-width, initial-scale=1">

<style>
pre {
	background-color: white;
	border: 1px grey solid;
	padding: 20px;
}
li {
	margin: 0 0 10px 0;
}
</style>

</head>
<body  link = "#736D6D" vlink = "#000000" alink = "#30ABE9">
<div id="wrapper_b">
	<div id="wrapper">
		<div id="lable_b">
			<div id="logo">
				<a href="../index.html">
				<!--<img src="../figures/yaz_lable.png" width="120px" height="70px">-->
				</a>
			</div>
		</div>

		<div id = "geneinfo_index">
			<div id="cor">
					<br></br>
					<p id = "genename" class = "mainH" ></p>
			</div>
		</div>

		<div id="content">
			
			<ul id="how-to">
				<a href="#upload"><li>How to upload my dataset</li></a>
				<a href="#use"><li>How to use or delete already uploaded dataset</li></a>
				<a href="#interface"><li>Description of tool interface</li></a>
				<a href="#single"><li>How to get an expression map for single ID</li></a>
				<a href="#multiple"><li>How to get an expression map for a list of IDs</li></a>
				<a href="#compare"><li>How to get a map of expression difference between two IDs</li></a>
				
			</ul>
			<h3 class='help-header' id="upload">How to upload my dataset</h3>
			<p>
				Click on the "UPLOAD A NEW DATASET" button on the main page. Below you will see a form for dataset uploading.
			</p>
			<p>
				In the first field you can browse ".tsv", ".xls" or ".xlsx" file with your dataset. This dataset must be Tab-separated (for ".tsv"-file) and be in format like:
			</p>
			<pre>ID	ID_1	ID_2	ID_3	ID_4	ID_5
ZFA_0005145	67	6	66	0	0
ZFA_0001076	890	77	34	1	90
ZFA_0000368	78	6	245	700	8
ZFA_0000403	56	7	0.2	890	0.4
ZFA_0000436	6	89	0	510	56</pre>
			<p>or:</p>
			<pre>ZFA_0000008	ZFA_0000107	ZFA_0000123	ZFA_0000403	ZFA_0000598
ID_1	400	11	12	0,2	0,5
ID_2	0,67	3	2	50	55
ID_3	43	56	523	0	0
ID_4	677	56	5	0,07	34
ID_5	123	567	580	1123	345</pre>
			<p>
				<b>ZFA</b> stands for ZebraFish Anatomy ontology CV-term. You can use ZFS (ZebraFish developmental Stage) as well. <a href="https://www.ebi.ac.uk/ols/ontologies/zfa" terget="_blank">Ontology Lookup Service</a> should help you to correspond particular tissue or developmantal stage name with ZFA or ZFS ID, respectively.
			</p>
			<p class="warning">
				There is no unique carp <i>(Cyprinus carpio)</i> Anatomy and Developmental stages Ontology, therefore we use Zebrafish Anatomy ontology for carp as well, since these species have no dramatic differencies in anatomy.
			</p>
			<p>
				Instead of browsing the file, if you check "paste your TSV-formatted data in a text-box", you will be able to paste your data in the same format into text-box, becoming visible under the checkbox. It might be usefull for small datasets.
			</p> 
			<p>
				The other fields describe attributes of uploading dataset and their functions are obvious.
			</p>
			<p class="message">
				Note! All fields are necessary to be filled. If you miss one or make a mistake you will see an alert message.
			</p>
			<p>
				As soon as you press "Submit" button, tool starts preprocessing your dataset. It may take from 15 sec to several minutes, depending on your dataset size. During this operation tool maps ZFAs and ZFSs of your dataset to such ones that tool can visualize.  
			</p>
			<p>
				Since preprocessing is done you will see a message, offering you to start your <a href="#interface">dataset visualization</a> ("Yes") or to stay on the current page ("No") to upload another dataset or to <a href="#use">use</a> previously uploaded dataset.
			</p>
			
			<h3 class='help-header' id="use">How to use or delete already uploaded dataset</h3>
			<p>
				Click on the "USE DATASET" button on the main page. Below you will see fields to select dataset or to delete dataset.
			</p>
			<p>
				To select dataset for visualisation simply select its name from select-box. You also may select predefined datasets from this select-box. Then press "Submit" button to proceed to the analysis.
			</p>
			<p>
				To delete dataset from tool select its name from select-box and press "Submit" button. So far you won't be able to use this dataset anymore.
			</p>
			<h3 class='help-header' id="interface">Description of tool interface</h3>
			<img src="../figures/interface.png" width="100%" style="border: 2px black solid">
			<p>
				Tool interface consists of 4 main parts:
			</p>
			<ol>
				<li><b>Navigation</b>. This part contains tabs to navigate between tool functions and the link to  the Home-page, where you can change dataset.</li>
				<li><b>Dataset information</b>. Here you can find current dataset properties, that were specified while dataset uploading and preprocessing. Here you can also view and save the list of all gene/protein IDs, presenting in current dataset, and the log of preprocessing with correspondence of ZFAs of current dataset to ZFAs that tool is able to visualize.</li>
				<li><b>Options</b>.  Using this options you can specify all available parameters of desirable picture. This part may contain the following fields:
				<ul>
					<li><u>ID</u>: here you should type ID of interest exactly as it is in uploaded table. Or click on "eye"-button (or on View-button of "The list of all gene/protein IDs" when you are using "Multiple maps"-tab) and select ID of interest from the list.</li>
					<li><u>Color scheme</u>: here you should select the color scheme from select-box or define your own one using special fields that appear after click on black triangle (you can type hexadecimal color code in these fields or pick a color from color picker).</li>
					<li><u>Unified gradient for</u>: here you may specify what normalization of axis you want. It is recommended to select "all proteins" if you want to visually compare several pictures and "each protein" in case if you want obtain one picture with best fit of values on axis.
					<p class="message">
				"Each protein" - tool will find maximal and minimal values of expression level for current ID and make gradient in this range. <br> "All proteins" - tool will find maximal and minimal values of expression level among all IDs in current dataset and make gradient in this range.
					</p></li>
					<li><u>Bins</u>: specify the number of colors that tool will divide the range of expression values into</li>
					<li><u>Scale</u>: scale for expression valuse (linear or Log10)</li>
					<li><u>Brain</u>: if checked, the tool will generate the largest picture of brain</li>
					<li><u>Developmental stages</u>: includes or excludes pictures of developmental stages</li>
				</ul></li>
				<li><b>Results.</b> Here will appear all pictures that tool generates. You are also able to save this pictures from this field in SVG or PDF format.</li>
			</ol>
			<h3 class='help-header' id="single">How to get an expression map for single ID</h3>
			<p>
				Select your dataset first. Click on "Single map" tab. To select ID of interest just simply type ID in the text-field or click on "eye"-button in "ID" field and choose needed ID from pop-up. Then specify options described in <a href="#interface">"Description of tool interface"</a>. 
			</p>
			
			<h3 class='help-header' id="multiple">How to get an expression map for a list of IDs</h3>
			<p>
				Select your dataset first. Click on "Multiple maps" tab. Type a list of IDs in "List of IDs" field or click on "View" button of "The list of all protein IDs available" and check IDs from pop-up. 
			</p>
			<p class="warning">
				Note! Each ID in list of IDs field must be on its own line.
			</p>
			<p>
				Then specify options described in <a href="#interface">"Description of tool interface"</a>. 
			</p>
			
			<h3 class='help-header' id="compare">How to get a map of expression difference between two IDs</h3>
			<p>
				Select your dataset first. Click on "Compare expression" tab. Select two IDs that you want to compare. To select ID of interest just simply type ID in the text-field or click on "eye"-button in "ID *" field and choose needed ID from pop-up. Then specify options described in <a href="#interface">"Description of tool interface"</a>. 
			</p>
		
		</div>

</div>


	<div id="footer">
	<div id = "footer_inner">
		<div class="nav">
			<ul>
				<li><a  href="../index.html">Tool</a></li>
				<li class="active"><a href="help.html">Help</a></li>
				<!--<li><a href="feedback.html">Feedback</a></li>-->
				<li  ><a href="about.html">About</a></li>
			</ul>
		</div>
		
		<p>&copy; Summer School in Bioinformatics, Leiden 2015-2016</p>
		<div id="site">
			<table align="center">
			<tr><td><a href="http://en.fbb.msu.ru/" target="_blank"><img src="../figures/msu.gif" height=70px width=70px></a></td>
			<td><a href="https://www.lumc.nl/org/proteomics-metabolomics/" target="_blank"><img src="../figures/lumc.jpg" height=70px width=70px></a></td></tr>
			</table>
		</div>
	</div>	
	</div>
</div>	
</body>
</html>
